doc/genstdlib.jl

```diff
@@ -3,7 +3,7 @@ module GenStdLib
 import Base.Docs: Binding, DocStr
 
 # Constants.
-const DOCSTRING_DIRECTIVE = r"^(.. (function|type|variable):: ).*"
+const DOCSTRING_DIRECTIVE = r"^(.. (function|type|variable|data):: ).*"
```

Comment:
Done in https://github.com/JuliaLang/julia/pull/17417/commits/9e0a4d2f06e29f104b8060b3aac71c2ba9bde1f4.